docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -W --keep-going #-n
```

Comment:
```suggestion
# W: Turn warnings into errors
# n: Run in nit-picky mode
SPHINXOPTS    = -W --keep-going #-n
```

Reply:
I'm trying to get in the habit of using verbose flag names when possible in scripts to make them easier to read for newcomers. Here Sphinx doesn't have verbose flags, so might not hurt to just add comments above.